docs/resources/dashboard.md

```diff
@@ -14,7 +14,7 @@ description: |-
 
 ```terraform
 resource "checkly_dashboard" "dashboard_1" {
-  custom_url      = "checkly"
+  custom_url      = "checkly" # Should be updated to a unique value. Otherwise, there will be a 409 conflict error.
```

Comment:
Just copying and pasting this example triggers a 409. Users will need to update this.